.github/workflows/lint.yml

```
@@ -38,6 +38,7 @@ jobs:
           version: v1.60
           args: --timeout 10m --out-${NO_FUTURE}format colored-line-number
           github-token: ${{ secrets.github_token }}
+          verify: false
```

actionlint (1.7.4)

input "verify" is not defined in action "golangci/golangci-lint-action@v6". available inputs are "args", "cache-invalidation-interval", "github-token", "install-mode", "only-new-issues", "problem-matchers", "skip-cache", "skip-save-cache", "version", "working-directory"

(action)